packages/ag-charts-community/src/chart/series/cartesian/scatterSeries.ts

```diff
@@ -314,7 +333,11 @@ export class ScatterSeries extends CartesianSeries<SeriesNodeDataContext<Scatter
         }
 
         const data = enabled ? nodeData : [];
-        return markerSelection.update(data);
+        return markerSelection.update(
+            data,
+            undefined,
```

Comment:
This was throwing some error in the line series when running tests, so I think Alex removed it. Not sure if it's been resolved now